unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/Tests/MapRendererShould.cs

```diff
@@ -118,7 +118,7 @@ public void DisplayParcelOfInterestIconsProperly()
         public void DisplayParcelOfInterestIconsProperly2()
         {
             var sceneInfo = new MinimapMetadata.MinimapSceneInfo();
-            sceneInfo.name = "important scene";
+            sceneInfo.name = "important scene 2";
```

Comment:
Do we need this?

Reply:
I don't why it was added - it left here from Juani. Maybe test was failing otherwise...